src/stores/game.ts

```diff
@@ -17,6 +17,8 @@ export default class GameStore {
   private _port: number | undefined;
 
   public constructor() {
+    makeObservable(this);
```

Comment:
https://mobx.js.org/enabling-decorators.html

Reply:
Tested and working.